AWS/EBS_Encryption/SCP/DenyUnencryptedEC2.json

```diff
@@ -1,13 +1,33 @@
 {
     "Version": "2012-10-17",
-    "Statement": [{
-        "Effect": "Deny",
-        "Action": ["ec2:CreateVolume", "ec2:RunInstances"],
```

Comment:
We want the policy to prevent creating unencrypted volume and turn on unencrypted instance

Reply:
Yes, I am simply suggesting that each API action be broken into its own statement, so you can be more granular with the descriptive explanation in the`Sid`:

```
        {
            "Sid": "NewUnencryptedVolumesCannotBeCreated",
            "Effect": "Deny",
            "Action": ["ec2:CreateVolume"],
            "Resource": "*",
            "Condition": {
                "Bool": {
                    "ec2:Encrypted": "false"
                }
            }
        },
        {
            "Sid": "InstancesWithUnencryptedVolumesCannotBeLaunched",
            "Effect": "Deny",
            "Action": ["ec2:RunInstances"],
            "Resource": "*",
            "Condition": {
                "Bool": {
                    "ec2:Encrypted": "false"
                }
            }
        },
        {
            "Sid": "EbsEncryptionByDefaultSettingCannotBeDisabled",
            "Effect": "Deny",
            "Action": ["ec2:DisableEbsEncryptionByDefault"],
            "Resource": "*"
        }
```